packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -339,25 +344,19 @@ export class Expression extends React.Component<Props, ExpressionState> {
     };
 
     // TODO(Nicole): I believe this is going away and won't be needed anymore
-    getGrammarTypeForPath: (inputPath: string) => string = (
-        inputPath: string,
-    ) => {
+    getGrammarTypeForPath(inputPath: FocusPath): string {
         /* c8 ignore next */
         return "expression";
-    };
+    }
 
-    setInputValue: (path: string, newValue: string, cb: () => void) => void = (
-        path: string,
-        newValue: string,
-        cb: () => void,
-    ) => {
+    setInputValue(path: FocusPath, newValue: string, cb: () => void) {
```

Comment:
This is an example of how conforming to the actual interface helps clean things up. Also, this widget was incorrectly typing `path` as a string, when it's really a `FocusPath` (array of strings - widget ids).

Reply:
I tried that, and it seems like TypeScript is not built to do that automatically. 

<img width="681" alt="image" src="https://github.com/user-attachments/assets/fa974900-016c-4e9c-ae39-d85fbb1c254e">